src/cpu/cpu_convolution_list.cpp

```diff
@@ -113,7 +115,7 @@ const std::map<conv_impl_key_t, std::vector<pd_create_f>> impl_list_map {
 #endif
         CPU_INSTANCE(gemm_convolution_fwd_t)
         CPU_INSTANCE(ref_convolution_fwd_t, f32)
-        CPU_INSTANCE(ref_fused_convolution_fwd_t)
+//        CPU_INSTANCE(ref_fused_convolution_fwd_t)
```

Comment:
If it is unused primitive lets put it under macro ENABLE_UNUSED_PRIM

Reply:
fixed, thanks